src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/DecXtractEventListener.java

```diff
@@ -0,0 +1,101 @@
+package de.uhd.ifi.se.decision.management.jira.extraction;
+
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.atlassian.event.api.EventListener;
+import com.atlassian.event.api.EventPublisher;
+import com.atlassian.jira.event.issue.IssueEvent;
+import com.atlassian.jira.event.type.EventType;
+import com.atlassian.plugin.spring.scanner.annotation.imports.JiraImport;
+
+import de.uhd.ifi.se.decision.management.jira.extraction.connector.ViewConnector;
+import de.uhd.ifi.se.decision.management.jira.extraction.model.impl.CommentImpl;
+import de.uhd.ifi.se.decision.management.jira.extraction.persistence.ActiveObjectsManager;
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.DecisionKnowledgeElementImpl;
+
+/**
+ * Triggers the decXtract related function when some changes to comments are
+ * made.
+ */
+@Component
+public class DecXtractEventListener implements InitializingBean, DisposableBean {
+
+	@JiraImport
+	private final EventPublisher eventPublisher;
+
+	private DecisionKnowledgeElement actionElement;
+
+	private String projectKey;
+
+	private IssueEvent issueEvent;
+
+	@Autowired
+	public DecXtractEventListener(@JiraImport EventPublisher eventPublisher) {
+		this.eventPublisher = eventPublisher;
+	}
+
+	/**
+	 * Called when the plugin has been enabled.
+	 * 
+	 * @throws Exception
+	 */
+	@Override
+	public void afterPropertiesSet() throws Exception {
+		eventPublisher.register(this);
+	}
+
+	/**
+	 * Called when the plugin is being disabled or removed.
+	 * 
+	 * @throws Exception
+	 */
+	@Override
+	public void destroy() throws Exception {
+		eventPublisher.unregister(this);
+	}
+
+	@EventListener
+	public void onIssueEvent(IssueEvent issueEvent) {
```

Comment:
@Jcl15 and @dekome , do think it would better to integrate the event listeners for webhook and comment events into one class because we have the onIssueEvent twice?